CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Feature
 
+- disable dragging for desktop devices to prevent link activation on drag @ThomasKindermann
```

Comment:
@ThomasKindermann minor: changelog entries always start with uppercase character.

Reply:
Difficult one... in fact, from the library point of view, it's a feature which we are disabling :)